src/main/webapp/test/mentors.test.js

```diff
@@ -0,0 +1,67 @@
+import {getMentors, createMentorElement, createEmailLink, createLocationHeader, createImg, createBioParagraph, createNameHeader, createGitHubLink, createInterestTagsDiv} from '../mentors.js';
+import '@testing-library/jest-dom/extend-expect';
+
+global.mailtouiApp = {run: () => {}};
+
+const mockMentor = {
+  interestTags: ['interest1', 'interest2'],
+  projects: [{respositoryID: 'mockRepoID', name: 'mockProjectName'}],
+  bio: 'This is the mock bio.',
+  email: 'samialves@google.com',
+  gitHubID: 'mockGitID',
+  image: '../images/dior.jpg',
+  location: 'Mockville, Mocktana',
+  name: 'Mocky the Mockster',
+};
+
+describe('Mentor card', () => {
+  it('is created', () => {
+    const mentorElement = createMentorElement(mockMentor);
+    expect(mentorElement).not.toBeNull();
+  });
+
+  it('has the correct image element', () => {
```

Comment:
I see that these all have a similar pattern of `has the correct <element>`. As a reader of these tests, it's hard for me to know what makes the element "correct". Consider being more explicit in these strings or in comments as to what makes each element correct versus not correct. I found [this comment from Bill](https://github.com/googleinterns/open-sesame/pull/142#discussion_r461744464) to be useful.